packages/core/docker/Dockerfile

```diff
@@ -61,6 +64,19 @@ RUN if [ "$DATABASE_URL" ]; then \
     unset DATABASE_URL; \
   fi
 
-EXPOSE 3000
+# This is for Drizzle studio
+EXPOSE 3002
+
+# This was not working
+# I need to keep container running to have Drizzle studio port open
+# Ex.:
+#  https://local.drizzle.studio?port=3002&host=0.0.0.0
+
+# CMD pnpm db:migrate
+
+# This entrypoint runs the migration
+ENTRYPOINT ["/entrypoint.sh"]
+
+# This is a hack to keep running the container
+CMD ["tail", "-f", "/dev/null"]
```

Comment:
I need help here. It works but this has to be different for production. 
Also when doing `docker compose stop core` it does not work

Reply:
# EXEC
`docker compose exec core pnpm db:studio`
![image](https://github.com/latitude-dev/latitude-llm/assets/49499/84d50563-19aa-468a-99a0-00fc375c3fbb)

With `compose run` it does not work